tools/podio-merge-files

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+"""podio-merge-files tool to merge any number of podio files into one"""
+
+import argparse
+import sys
+import podio
+import podio.root_io
+from podio import reading
+
+parser = argparse.ArgumentParser(
+    description="Merge any number of podio files into one, can merge TTree and RNTuple files"
+)
+
+parser.add_argument("--output-file", help="name of the output file", required=True)
+parser.add_argument("files", nargs="+", help="which files to merge")
+parser.add_argument(
+    "--metadata",
+    choices=["none", "all", "first"],
+    default="first",
+    help="metadata to include in the output file, default: "
+    "only the one from the first file, other options: all files, none",
+)
+args = parser.parse_args()
+
+all_files = set()
+for f in args.files:
+    if f in all_files:
+        raise ValueError(f"File {f} is present more than once in the input list")
+    all_files.add(f)
```

Comment:
Maybe would be better to keep the order of the files as given in the argument list? `set()` is not insertion ordered

Reply:
Ahh, you don't use all_files later on.